src/nwb_datajoint/common/common_spikesorting.py

```diff
@@ -559,14 +555,13 @@ def make(self, key: dict):
                                                                                    sort_interval)
         AnalysisNwbfile().add(key['nwb_file_name'], key['analysis_file_name'])       
         
-        key['sorting_id'] = 'S_'+str(uuid.uuid4())[:8]
-
-        # add sorting to Sorting
-        Sortings.insert1({'nwb_file_name': key['nwb_file_name'],
-                          'recording_id': key['recording_id'],
-                          'sorting_id': key['sorting_id'],
-                          'sorting_path': key['sorting_path'],
-                          'parent_sorting_id': ''}, skip_duplicates=True)
+        sorting_key = (self & key).proj().fetch1()
+        sorting_key['sorting_id'] = 'S_'+str(uuid.uuid4())[:8]
+        sorting_key['sorting_path'] = key['sorting_path']
+        sorting_key['parent_sorting_id'] = ''
+        
+        # add sorting to SortingID
+        SortingID.insert1(sorting_key, skip_duplicates=True)
```

Comment:
Is this proper datajoint practice, to insert an item into another table as part of the make() function of a table?

Also, if the sorting id is random, how is the user going to be able to find it later on?